pkgs/development/go-modules/generic/default.nix

```diff
@@ -184,8 +187,8 @@ go.stdenv.mkDerivation (
     runHook preInstall
 
     mkdir -p $bin
-    dir="$NIX_BUILD_TOP/go/bin"
-    [ -e "$dir" ] && cp -r $dir $bin
+    dir="$NIX_BUILD_TOP/go/bin/${gobin_prefix}"
```

Comment:
I wonder if we should remove the prefix already in the `buildPhase` in case someone breaks it by overriding `installPhase`?